### PR TITLE
bug: add missing inputs for release workflow

### PR DESCRIPTION
**Pull request by acedeywin**
## Description

Adds the parameters used in the QA release workflow to line up with the ones used by custom_helpline.

### Checklist
- [N/A] Corresponding issue has been opened
- [N/A] New tests added

### Related Issues
Fixes N/A

### CODE REVIEW

File: .github/workflows/serverless-pre-release-qa.yml [hunk 1]
```diff
@@ -26,6 +26,27 @@ on:
       title:
         description: Release title - The title that will be given to this pre-release.
         required: true
+      helpline_code:
```
Comment (stephenhand) [inline .github/workflows/serverless-pre-release-qa.yml:29]: We don't need extra options on the workflow, we just need to pass `environment: development` to the call to `custom_helpline.yml` on line 35

**Review by stephenhand — CHANGES_REQUESTED** [review]

File: .github/workflows/serverless-pre-release-qa.yml (hunk 1 above)
Comment (acedeywin) [reply .github/workflows/serverless-pre-release-qa.yml]: @stephenhand  - Oh, I see. Sorry for my misunderstanding. It's done.

Comment (acedeywin) [reply .github/workflows/serverless-pre-release-qa.yml]: @stephenhand  - Oh, I see. Sorry for my misunderstanding. It's done.

**Review by stephenhand — APPROVED** [review]
Looks good now

**Pull request merged**